types/entropy__winston-elasticsearch-apm/index.d.ts

```diff
@@ -0,0 +1,19 @@
+// Type definitions for @entropy/winston-elasticsearch-apm 1.0
+// Project: https://gitlab.entropy.cc/entropy/winston-elasticsearch-apm
+// Definitions by: Michael Vasyliv <https://github.com/michael-vasyliv>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import Agent = require('elastic-apm-node');
+import TransportStream = require('winston-transport');
+
+export interface ElasticsearchApmOptions extends TransportStream.TransportStreamOptions {
+    apm: typeof Agent;
+}
+
+declare class ElasticsearchApm extends TransportStream {
+    constructor(opts: ElasticsearchApmOptions);
+
+    log(info: any, next: () => void): any;
+}
+
+export default ElasticsearchApm;
```

Comment:
The `cjs` usage of this package in the `README` is

`var ElasticsearchApm = require('@entropy/winston-elasticsearch-apm');` - please use an `export =` here to support that, and don't disable the lint rules requesting you use an `export =`, and then turn on `esModuleInterop` in the `tsconfig` if you'd still like to use `default` imports in the tests.

Reply:
@weswigham done